src/components/RulesSelector.jsx

```diff
@@ -27,19 +29,14 @@ const makeListItems = (filters, onRuleSelected) => (
 const RulesSelector = ({ filters, onRuleSelected }) => {
   const listItems = makeListItems(filters, onRuleSelected);
   return (
-    <div>
+    <div className="rule-selector-container">
```

Comment:
I meant to say that the elements in AppBody are using CSS id selectors, not classes

In this case, we're not required to use CSS selectors to apply styles to the buttons, although I think it's more appropriate to do so in this case for a couple of reasons. That said, I can revert to using our previous method.